data_safe_haven/resources/workspace/ansible/desired_state.yaml

```diff
@@ -100,6 +100,36 @@
         validate: sshd -T -f %s
       notify: Restart sshd
 
+    - name: Copy xrdp settings
+      ansible.builtin.copy:
+        src: etc/xrdp/xrdp.ini
+        dest: /etc/xrdp/xrdp.ini
+        mode: '0644'
+
+    - name: Copy xrdp logo
+      ansible.builtin.copy:
+        src: usr/share/xrdp/dsh_logo_240x140_256color.bmp
+        dest: /usr/share/xrdp/dsh_logo_240x140_256color.bmp
```

Comment:
We should write this to another location, maybe `/var` or `/usr/local/share`. `/usr/share` is managed by `apt`.